eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -235,7 +235,7 @@ stages:
                           # we override the regular script path because we have cloned the build tools repo as a separate artifact.
                           ScriptPath: 'eng/common/scripts/copy-docs-to-blobstorage.ps1'
 
-          - ${{if ne(artifact.options.skipUpdatePackageVersion, 'true')}}:
+          - ${{if ne(artifact.skipUpdatePackageVersion, 'true')}}:
             - deployment: UpdatePackageVersion
               displayName: "Update Package Version"
               condition: and(succeeded(), ne(variables['Skip.UpdatePackageVersion'], 'true'))
```

Comment:
In other PRs, we swapped the optional `Integration` publish to the dev feed under `skipPublishDevFeed`. 

![image](https://user-images.githubusercontent.com/45376673/93123852-95099700-f67d-11ea-8659-716c865c288c.png)

That screenshot is from JS. I kinda figure we should update everything if we're doing a triggering update.

Reply:
I search all nested structure in the release yml. That's all we need to update. Do you suggest anything else?

Reply:
.NET is not having artifact level publish dev feed. So ignore in .net.

Reply:
Yeah we can ignore it for the dev publish for now until a time were we might need it.